tests/e2e/visual/targets/targets-overview.wdio-spec.js

```diff
@@ -0,0 +1,694 @@
+  });
+  const pregnantWoman17 = personFactory.build({
+    patient_id: 'test_woman_17',
+    date_of_birth: pregnantWomanDateOfBirth,
+    parent: { _id: healthCenter._id, parent: healthCenter.parent }
+  });
+  const pregnantWoman18 = personFactory.build({
+    patient_id: 'test_woman_18',
+    date_of_birth: pregnantWomanDateOfBirth,
+    parent: { _id: healthCenter._id, parent: healthCenter.parent }
+  });
+  before(async () => {
+    await utils.saveDocs([...places.values(), pregnantWoman , pregnantWoman2, pregnantWoman3 , pregnantWoman4, pregnantWoman5, pregnantWoman6, pregnantWoman7, pregnantWoman8, pregnantWoman9, pregnantWoman10, pregnantWoman11, pregnantWoman12, pregnantWoman13, pregnantWoman14, pregnantWoman15, pregnantWoman16, pregnantWoman17, pregnantWoman18]);
+    await utils.createUsers([offlineUser]);
+    await loginPage.login(offlineUser);
+  });
+  //form creations
+  it('New  pregnancy form creation', async () => {
+    //pregnacy form creation
+    const edd = moment().add(8, 'days');
+    const nextANCVisit = moment().add(2, 'day');
+
+    await commonPage.goToPeople(pregnantWoman._id);
+    await commonPage.openFastActionReport('pregnancy');
+    await pregnancyForm.submitDefaultPregnancy(false);
+
+    const summaryTexts = [
+      pregnantWoman.name,
+      '38', //weeks pregnant
+      edd.format('D MMM, YYYY'),
+      'Previous miscarriages or stillbirths',
+      'Previous difficulties in childbirth',
+      'Has delivered four or more children',
+      'Last baby born less than one year ago',
+      'Heart condition',
+      'Asthma',
+      'High blood pressure',
+      'Diabetes',
+      'Vaginal bleeding',
+      'Fits',
+      'Severe abdominal pain',
+      'Severe headache',
+      'Very pale',
+      'Fever',
+      'Reduced or no fetal movements',
+      'Breaking of water',
+      'Getting tired easily',
+      'Swelling of face and hands',
+      'Breathlessness'
+    ];
+
+    await commonEnketoPage.validateSummaryReport(summaryTexts);
+    await genericForm.submitForm();
+    //pregnancy visit form creation
+    await commonPage.openFastActionReport('pregnancy_home_visit');
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+    await commonPage.waitForPageLoaded();
+    await commonPage.goToPeople(pregnantWoman._id);
+    await commonPage.openFastActionReport('pregnancy_home_visit');
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+    await commonPage.openFastActionReport('pregnancy_home_visit');
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+
+
+  });
+  it('New Pregnancy form creation and Delivery in facility', async () => {
+    //pregnacy form creation
+    await commonPage.waitForPageLoaded();
+    await commonPage.goToPeople(pregnantWoman2._id);
+    await commonPage.openFastActionReport('pregnancy');
+    await pregnancyForm.submitDefaultPregnancy();
+    await commonPage.openFastActionReport('delivery');
```

Comment:
Can you please try using one of the pregnant women generated by `createHierarchy` to create the delivery report, instead of generating a new pregnancy again?